.github/CONTRIBUTING.md

```diff
@@ -177,7 +189,7 @@ Additional references:
   Company = "Microsoft Corporation"
   Copyright = "Copyright (c) Microsoft Corporation. All rights reserved."
   ```
-
+  is at the top.  
```

Comment:
Please make sure that there is a newline before and after any header (the spelling task will fail if there is not)